packages/tools/eslint-config-custom/package.json

```diff
@@ -22,5 +22,6 @@
   },
   "publishConfig": {
     "access": "public"
-  }
+  },
+  "types": "./dist/index.d.ts"
```

Comment:
nit: feels like this field should be located near where other fields like this are, e.g. near `files`

Reply:
Oh, this should not be here at all.